toptal/picasso
# chore: [FX-2495] Remove critical vulnerabilities from picasso

[FX-2495](https://toptal-core.atlassian.net/browse/FX-2495?atlOrigin=eyJpIjoiNDU1ODY3MWU3MjE1NDIxODk3Y2Q1YzEyNjI3MzJhNjYiLCJwIjoiaiJ9)

### Description

- Remove critical vulnerabilities from Picasso

### How to test

- Run `yarn audit --level critical` in Picasso repo and we should see 0 critical vulnerabilities:
![Screenshot from 2022-02-17 19-22-19](https://user-images.githubusercontent.com/72510037/154513233-b7445507-6aae-4f8d-a0ef-58c0a5c689a8.png)


### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
